docs/system-models.rst

```diff
@@ -53,16 +54,123 @@ necessary to specify a particular storage systems.
 Health Parameters
 -----------------
 
-The health parameters for a battery are defined as a subclass of ``HealthVariable``.
+The health parameters for a battery are defined as a subclass of :class:`~moirae.models.base.HealthVariable`.
 As with the transient states, health parameters are specific to the type of battery.
 Unlike transient states, the way they are defined allows a hierarchical structure
 and the ability annotate which parameters are treated as updatable.
 
-.. note:: TODO, figure out how much documentation to copy over from the class docstring
+Consider the following parameter set as an example
 
-Models
-------
+.. code-block:: python
+
+    class Resistance(HealthVariable):
```

Comment:
Should we make it clear that this is not how the implemented `Resistance` works?

Reply:
I don't think that's necessary given how different it is from the resistance we have now